Sources/TokamakDemo/SliderDemo.swift

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import TokamakShim
+
+struct SliderDemo: View {
+  @State private var value: Double = 0
+
+  var body: some View {
+    VStack {
+      Slider(value: $value, in: 0...1, step: 0.1, minimumValueLabel: Text("min"), maximumValueLabel: Text("max")) {
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakDemo/SliderDemo.swift;
  Line: 22;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 115 characters (`line_length`)